src/commons/eventsourcing/mod.rs

```diff
@@ -294,65 +337,69 @@ mod tests {
 
     #[test]
     fn event_sourcing_framework() {
-        test_under_tmp(|data_dir| {
-            let counter = Arc::new(EventCounter::default());
-            let storage_uri = storage_uri_from_data_dir(&data_dir).unwrap();
-            let mut manager = AggregateStore::<Person>::create(&storage_uri, segment!("person")).unwrap();
-            manager.add_post_save_listener(counter.clone());
+        // crate::test::test_under_tmp(|data_dir| {
```

Comment:
Is there a reason to leave this here? Were you done and this is can be removed, or where you in the middle of something and it's actually still relevant?

Reply:
Ah I guess this is obsoleted by the KVX changes and should be removed?